website/src/css/customTheme.scss

```diff
@@ -64,12 +66,47 @@ html[data-theme="dark"] {
   }
 }
 
+::marker {
+  color: var(--ifm-font-color-secondary);
+}
+
+/* Font */
+
+@font-face {
+  font-family: "Optimistic Display";
+  src: url("https://facebookmicrosites.github.io/design/public/fonts/OptimisticDisplayLight-199be98cf48e5b4c688356b08a02362c.woff2")
```

Comment:
@rachelnabors  A non-blocking question -- is there a better host for these fonts? @Simek and I were wondering if there is a plan to host these on a CDN somewhere